orc8r/cloud/go/services/accessd/storage/storage_blobstore.go

```diff
@@ -57,7 +57,11 @@ func (a *accessdBlobstore) ListAllIdentity() ([]*protos.Identity, error) {
 		return nil, status.Errorf(codes.Internal, "failed to list keys: %s", err)
 	}
 
-	tks := blobstore.GetTKsFromKeys(AccessdDefaultType, idHashes)
+	if len(idHashes) == 0 {
+		return make([]*protos.Identity, 0), store.Commit()
```

Comment:
Move+rename the `ret` variable below to the top of this method as `var ids []*protos.Identity`. Then for this early return you can just `return ids, store.Commit()`